addons/mail/controllers/discuss/public_page.py

```diff
@@ -51,11 +51,14 @@ def discuss_channel_invitation(self, channel_id, invitation_token):
 
     @http.route("/discuss/channel/<int:channel_id>", methods=["GET"], type="http", auth="public")
     @add_guest_to_context
-    def discuss_channel(self, channel_id):
+    def discuss_channel(self, channel_id, highlight_message_id=None):
```

Comment:
Cannot be done in 18 (breaks overrides as they are going to receive an unexpected parameter).